pkg/timesync/timesync.go

```diff
@@ -171,7 +171,6 @@ func (t *TimeSync) QueryNTP() error {
 		if err == nil {
 			break
 		}
-		log.Warnw("error querying NTP server", "attempt", i+1, zap.Error(err))
 	}
 	if err != nil {
```

Comment:
I think this line should be moved after the final error check to ensure this error is logged at least once regardless of what the callers of `QueryNTP()` do.

Reply:
I'd disagree, QueryNTP correctly returns an error in case of failure, it is up to the caller to log it, and it does [here](https://github.com/Metrika-Inc/agent/pull/67/files#diff-c0670d2e7dc7093c3e1312515a4b8fec2da25b9543e0be5a0a9d8662d5f1f415R119)